scripts/deploy

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+set -e
+
+deploy_new=""
+no_cache=""
+
+function usage() {
+    echo -n "Usage: $(basename "${0}") [OPTION]
+Deploys a Tilegarden instance to AWS by building docker containers first
+Options:
+    --help        Show usage
+    --new        Deploy as a new lambda
+    --no-cache    Don't use cache when building containers
+"
+    exit 0
+}
+
+function main() {
+    docker-compose build terraform
+
+	if [ "${no_cache}" == "--no-cache" ]; then
+    	docker-compose build --no-cache tiler
+    else
+    	docker-compose build tiler
+    fi
+
+    if ./scripts/publish "${deploy_new}" ; then
+        :
+    else
+    	echo ""
+        echo "Did you maybe mean './scripts/deploy --new'?"
```

Comment:
I cleared by `AWS_REGION` to see what would happen and ran `./scripts/deploy --new`.  It did this:
```
AWS region is missing. please specify with --region
error An unexpected error occurred: "Command failed.
Exit code: 1
Command: sh
Arguments: -c yarn transpile && claudia create --api-module bin/api --name ${PROJECT_NAME} --region ${LAMBDA_REGION} ${LAMBDA_ROLE:+--role ${LAMBDA_ROLE}} ${LAMBDA_TIMEOUT:+--timeout ${LAMBDA_TIMEOUT}} ${LAMBDA_MEMORY:+--memory ${LAMBDA_MEMORY}} ${LAMBDA_SECURITY_GROUPS:+--security-group-ids ${LAMBDA_SECURITY_GROUPS}} ${LAMBDA_SUBNETS:+--subnet-ids ${LAMBDA_SUBNETS}} && yarn parse-id
Directory: /home/tiler
Output:
".
info If you think this is a bug, please open a bug report with the information provided in "/home/tiler/yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

Did you maybe mean './scripts/deploy --new'?
```
We probably shouldn't suggest this possibility for all errors.  At the least, we shouldn't suggest it when the `--new` option was in fact given.